pkgs/development/python-modules/cron-descriptor/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pep8
+, pep8-naming
+, flake8
+}:
+
+buildPythonPackage rec {
+  pname = "cron_descriptor";
+  version = "1.2.24";
+
+  # Fetch source from GitHub, because pypi releases do not contain tests.
+  src = fetchFromGitHub {
+    owner = "Salamek";
+    repo = "cron-descriptor";
+    rev = version;
+    sha256 = "sha256-Gf7n8OiFuaN+8MqsXSg9RBPh2gXfPgjJ4xeuinGYKMw=";
+  };
+
+  checkInputs = [
+    pep8
+    flake8
+    pep8-naming
```

Comment:
We don't care at all about linting tests and should not execute them because they easily break eg on flake8 updates.

but I saw that there are tests we should execute explicitly https://github.com/Salamek/cron-descriptor/tree/master/tests

Reply:
It's implicitly running "python setup.py test" which runs the tests you've linked as well. If i naively remove the linting test inputs, setuptools tries to download them during build time and fails ofc. So the easy way is to just let them run. An alternative could be to try running the tests with pytest to use the existing disableTests facilities in that hook.  Do you think not running linting-tests is worth the effort?

Reply:
Yes, it is worth the effort to patch them out. Linters should not cause mass rebuilds and updating eg flake8 should not break healthchecks because they added some new rule. This makes also pythonPackages maintenance a lot easier and gives us way less surprises.

Reply:
The linters weren't even run, just installed to keep setup.py happy :) But I agree that it doesn't make sense to keep them in if they aren't really used. So patched out now.